query-service/build.gradle.kts

```diff
@@ -8,14 +8,14 @@ plugins {
 dependencies {
   implementation(project(":query-service-impl"))
   implementation("org.hypertrace.core.grpcutils:grpc-server-utils:0.3.0")
-  implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.18")
+  implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.21")
   implementation("io.grpc:grpc-netty:1.33.0")
   implementation("org.slf4j:slf4j-api:1.7.30")
 
-  runtimeOnly("io.netty:netty-codec-http2:4.1.54.Final") {
+  runtimeOnly("io.netty:netty-codec-http2:4.1.59.Final") {
```

Comment:
These are incorrectly declared - they should be inside a constraints block

Reply:
moved